tests/php/PHPMessDetectorBearTest.py

```diff
@@ -16,7 +16,8 @@ def get_testfile_path(name):
 
 
 def load_testfile(name):
-    return open(get_testfile_path(name)).readlines()
+    with open(get_testfile_path(name)) as f1:
```

Comment:
Can you please rename `f1` to something meaningful ? May be `file` ?

Reply:
@shreyans800755 sure 👍